<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>18</number>
       </property>
       <item>
        <widget class="QLabel" name="fileName">
         <property name="text">
          <string>文件名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileName_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="fileType">
         <property name="text">
          <string>文件类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileType_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>49</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>保存当前数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>关闭当前文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="New"/>
    <addaction name="Open"/>
    <addaction name="Save"/>
    <addaction name="separator"/>
    <addaction name="Quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="Explain"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="New">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/resource/new.png</normaloff>:/icon/resource/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="Open">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/resource/folder-new.png</normaloff>:/icon/resource/folder-new.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="Save">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/resource/save.png</normaloff>:/icon/resource/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="Quit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/resource/quit.jpg</normaloff>:/icon/resource/quit.jpg</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;Q)</string>
   </property>
  </action>
  <action name="Explain">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/resource/explain.jpg</normaloff>:/icon/resource/explain.jpg</iconset>
   </property>
   <property name="text">
    <string>说明</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
